app/components/profile/picture_component.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Profile::PictureComponent < ViewComponent::Base
+  JOB_SEARCH_COLOR_CLASSES = {
+    'not_job_searching' => 'bg-red-800 text-red-50',
+    'open_to_opportunities' => 'bg-amber-800 text-amber-50',
+    'open_to_work' => 'bg-green-800 text-green-50'
```

Comment:
Intentionally not being as "programmatic" about this. Tailwind needs the full class name for each utility or else that utility will be caught in its "purging" -- where Tailwind goes through and removes unused styles as a way to control bundle size.